config/devices.txt.example

```diff
@@ -1,3 +1,3 @@
 # For standalone mode, enter list of badges MAC addresses
-#<device> <comment / device description>
+#<device> <comment / device description> <badge_id> <project_id>
```

Comment:
I think it is better to keep the comment as the last field, since it may contain spaces.